model/src/main/proto/topic.proto

```diff
@@ -244,11 +244,20 @@ enum ChapterPlayState {
   // prerequisites.
   NOT_PLAYABLE_MISSING_PREREQUISITES = 2;
 
-  // The chapter has been started but not completed by the player.
-  STARTED_NOT_COMPLETED = 3;
+  // The chapter has been started but not completed by the player and the progress made in the
+  // chapter is saved.
+  IN_PROGRESS_SAVED = 3;
```

Comment:
We can't change the meaning if existing values--they need to be preserved or migrated to a reasonable replacement.

To that end, I think you want to keep the same ordinals but just add the new enums that we're going to use (no migration should be necessary).

Reply:
done